### ISSUE TITLE
Plugin is incompatible with Packer 1.7

### ISSUE THREAD

**Issue by UnsafePointer** [veertuinc/packer-builder-veertu-anka#67]
Please update the README file to make this more clear. 

```Bash
% packer build base.json
packer build base.json
Error: Failed to initialize build "veertu-anka"

error initializing builder 'veertu-anka': The protocol of this plugin (protocol
version 4 and lower) was deprecated, please use a newer version of this
plugin.Or use an older version of Packer (pre 1.7) with this plugin.
```

It seems that there are some plugin API changes in Packer 1.7, see https://github.com/hashicorp/packer/blob/master/CHANGELOG.md#backwards-incompatibilities.

**Comment by NorseGaud** [comment]
Thank you for reporting! We are working on a version to address this right now. It should
Be out in a day or two. I'll post the link here once it's available.

**Comment by NorseGaud** [comment]
Available in 1.8.0
